saleor/checkout/utils.py

```diff
@@ -688,3 +702,19 @@ def validate_variants_in_checkout_lines(lines: Iterable["CheckoutLineInfo"]):
                 )
             }
         )
+
+
+def call_payment_refund_or_void(
+    checkout_info: "CheckoutInfo",
+    payment: Optional[Payment],
+    manager: PluginsManager,
+):
+    if not payment:
+        return
+
+    if len(get_active_payments(checkout_info.checkout)) > 1:
```

Comment:
Shouldn't be called or all payments?

Reply:
This condition has been now changed not to cancel partial payments.